app/src/main/res/layout/multiple_choice_interaction_items.xml

```diff
@@ -26,7 +27,6 @@
       android:layout_width="wrap_content"
       android:layout_height="wrap_content"
       android:clickable="false"
-      android:padding="4dp"
```

Comment:
**Comment 1**
The Radio/Checkbox automatically contains `4dp` padding and therefore removed this padding.